tests/src/tracing/common/Assert.cs

```diff
@@ -4,36 +4,36 @@ namespace Tracing.Tests.Common
 {
     public static class Assert
     {
-        public static void Equal<T>(T left, T right) where T : IEquatable<T>
+        public static void Equal<T>(string name, T left, T right) where T : IEquatable<T>
```

Comment:
good idea to add the name to know what was wrong 👍 

not related directly to this change: xUnit has a very nice convention for naming of the `Assert` method arguments. These are `expected` and `actual`.  So when somebody else is reading the test it's more easy to understand what was the expected value. And the optional message comes as last, so it's `Equal(T expected, T actual, string message = null)`

Reply:
Thanks Adam.  Will keep that in mind as I add more tests.